tests/pkg_cryptoauthlib_internal-tests/main.c

```diff
@@ -26,9 +26,12 @@ int main(void)
     if (ATCA_DEVTYPE == ATECC608A) {
         atca_run_cmd("608");
     }
-    else {
+    else if (ATCA_DEVTYPE == ATECC508A) {
         atca_run_cmd("508");
     }
+    else {
+        printf("This device is currently not supported.");
```

Comment:
Can you still run the "unit" command with an unknown device, or should we rather abort here?

Reply:
You're right, we should abort. I'll fix that.

Reply:
You can squash directly.